content/es/docs/concepts/components.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Componentes
+description: Los principales componentes que forman OpenTelemetry
+aliases: [data-collection]
+weight: 20
+default_lang_commit: 9b7da35fd7abd77d867177902b36d95e5f322182
+---
+
+OpenTelemetry actualmente está compuesto por varios componentes principales:
+
+- [Especificación](#especificación)
+- [Collector](#collector)
+- [Implementaciones de API y SDK específicas del lenguaje](#implementaciones-de-api-y-sdk-específicas-del-lenguaje)
+  - [Librerías de Instrumentación](#librerías-de-instrumentación)
+  - [Exportadores](#exportadores)
+  - [Instrumentación sin código](#instrumentación-sin-código)
+  - [Detectores de Recursos](#detectores-de-recursos)
+  - [Propagadores entre servicios](#propagadores-entre-servicios)
+  - [Muestreadores](#muestreadores)
+- [Operador de Kubernetes](#operador-de-kubernetes)
+- [Elementos de Función como Servicio](#elementos-de-función-como-servicio)
```

Comment:
Check https://opentelemetry.io/docs/contributing/localization/#heading-anchors to make sure the PR is compliant.